### ISSUE TITLE
Installation issues on Jetpack 4.5 [L4T 32.5.2]

### ISSUE THREAD

**Issue by Kaapeine** [mzahana/jetson_svo_docker#3]
Has anyone got this to work with this system configuration? This is the error I'm seeing: 

> Docker: Error response from daemon: failed to create shim: OCI runtime create failed: container_linux.go:380: starting container process caused: error

This was during the ROS Melodic base image installation. People on the NVIDIA forums [suggest](https://forums.developer.nvidia.com/t/docker-error-response-from-daemon-failed-to-create-shim-oci-runtime-create-failed-container-linux-go-starting-container-process-caused-error/197663) downgrading but that would brick other containers I need. Does anybody have a solution for this?

(I was seeing another error before this, 'Manifest not found', and I was able to get around it by changing the L4T version passed to the  base image installation script to 32.5.0)

**Comment by mzahana** [comment]
Hi @Kaapeine 

I am not sure if the following suggestion will fix your issue, but you can try.

Try to replace the [scripts/docker_base.sh](https://github.com/mzahana/jetson_svo_docker/blob/81e5b9eb6c58209f0c3c6b233eb34f17d299a91b/scripts/docker_base.sh), with a more recent one found [here](https://github.com/dusty-nv/jetson-containers/blob/master/scripts/docker_base.sh). Then, remove the docker container/image and build again.

**Comment by Kaapeine** [comment]
> Owner

Thanks for the suggestion. I tried replacing the script with the one you linked but it didn't solve my issue. I saw the same error so I sorta gave up for a while because I can't downgrade Jetpack right now.

Coming back to it now, I was able to get it working on my Jetpack version by manually going through the steps in your SVO dockerfile. I created another container and followed the steps there, and copied the patches in this repo as needed. And, I finally got it work! Thank you for the patches!
